src/att.cpp

```diff
@@ -128,7 +128,7 @@ data_BEATRPT_REND Att::StrToBeatrptRend(const std::string &value, bool logWarnin
 std::string Att::BulgeToStr(const data_BULGE &data) const
 {
     std::ostringstream ss;
-    for (int i = 0; i < data.size(); ++i) {
+    for (size_t i = 0; i < data.size(); ++i) {
```

Comment:
For consistency do `(int)data.size()` instead

Reply:
For consistency I changed it to `size_t`. Or we change all other occurrences in `att.cpp`.

Reply:
Yes, change them.